src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -251,6 +251,7 @@ private function addFirewallsSection(ArrayNodeDefinition $rootNode, array $facto
                     ->scalarNode('path')->defaultValue('/logout')->end()
                     ->scalarNode('target')->defaultValue('/')->end()
                     ->booleanNode('invalidate_session')->defaultTrue()->end()
+                    ->scalarNode('clear_site_data')->defaultValue('off')->end()
```

Comment:
why don't we use an array node with enumerated possible values?

Reply:
Good idea, didn't know that way, fixed!